packages/node_modules/sample-browser-auth-implicit/app.js

```diff
@@ -0,0 +1,47 @@
+/* eslint-env browser */
+/* global ciscospark */
+/* eslint-disable camelcase */
+/* eslint-disable max-nested-callbacks */
+/* eslint-disable no-alert */
+/* eslint-disable no-console */
+/* eslint-disable require-jsdoc */
+
+// Declare some globals that we'll need throughout
+const spark = ciscospark.init();
+
+spark.once(`ready`, () => {
+
+  // Now, let's set up the event listener for the Authenticate Button
+  document.getElementById(`authorization`).addEventListener(`submit`, (event) => {
+    // let's make sure we don't reload the page when we submit the form
+    event.preventDefault();
+
+    if (spark.canAuthorize) {
+      // if already authenticated, logout on click
+      spark.logout();
+    }
+    else {
+      // initiate the login sequence if not authenticated
+      spark.authorization.initiateLogin();
+    }
+
+  });
+
+  if (spark.canAuthorize) {
+    // Authorization is successful
+
+    // your app logic goes here
+
+    // Change Authentication status to `Authenticated`
+    const authStatus = document.getElementById(`authentication-status`);
+    authStatus.innerHTML = `Authenticated`;
+    authStatus.style = `color: green`;
+
+    document.getElementById(`authBtn`).value = `logout`;
```

Comment:
let's use separate buttons instead of changing their value. The goal of the samples is explicitly to not be fancy.